README.md

```diff
@@ -1,101 +1,127 @@
 [![Build Status](https://travis-ci.org/space-concordia-robotics/robotics-prototype.svg?branch=master)](https://travis-ci.org/space-concordia-robotics/robotics-prototype)
 
 # robotics-prototype
-This repo contains the Robotics software team code. For a quick primer on our workflow using git, [CLICK HERE :)](https://github.com/space-concordia-robotics/robotics-prototype/wiki/Git-Workflow-and-Conventions)
+This repo contains the Robotics software team code.
 
 ## Contributing and Development Environment Instructions
 
 Firstly, this project is built in Python 3.6+ and JavaScript (ES6). You need to have a version of Python installed that is 3.6+. Make sure that whenever you use `python`, `python3` or `python3.6` or whatever later on meets this requirement.
 
 Secondly, it is imperative you use a virtual env (instead of your system Python) to use/contribute to the project, else things could get messy.
+### Prerequisites
+Make sure you are using Ubuntu 16.04 (The ROS distribution we use doesn't support anything newer than 16.04).
 
-### Setup [virtualenv](https://docs.python.org/3.6/library/venv.html#modulevenvhttps://virtualenv.pypa.io/en/stable/userguide/)
-Navigate to the projects root directory (`cd ~/.../robotics-prototype`) and create the virtual environment):
-Make sure you supply a path to a Python 3.6.x+ binary. Make sure to use backticks (not single quotes) for `which python3.6`:
+Make sure the following are installed
 ```
-$ virtualenv -p `which python3.6` venv
+$ sudo apt install python-pip
+$ sudo apt install virtualenv
+$ sudo apt install git
 ```
-**Note**: Your actual alias for calling Python might be different (e.g. `python`, `py`, `py3`, etc). You can ensure whichever it might be is version 3.6+ by typing `which python` in Linux/Mac (bash/zsh).
-
-You should see a new directory named `venv` inside the root directory. We can now activate this virtual environment:
-Linux/Mac (bash/zsh):
+You will also need python3.6, which is by default not in Ubuntu 16.04's APT's available packages.
+#### Installing python3.6
 ```
-$ source venv/bin/activate
+$ sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
I believe it is not required to add a ppa to install python.

Have you tried just `sudo apt-get install python3`?

Reply:
oh, ppa thing is for just python3.6, josh told me we should really stick with the newest one

Reply:
I use python3.6 and I have never added this ppa

Reply:
It's because you have to do `sudo apt-get install python3` not 3.6, to get the latest

Reply:
![image](https://user-images.githubusercontent.com/18620025/65841808-198bca00-e2f4-11e9-9c13-c45e4c7d201c.png)

Reply:
@vashmata @PeterGhimself